packages/pods-core/src/builtin/GraphvizPod.ts

```diff
@@ -128,7 +128,11 @@ export class GraphvizExportPod extends ExportPod<GraphvizExportConfig> {
 
     // verify dest exist
     const podDstPath = dest.fsPath;
-    fs.ensureDirSync(path.dirname(podDstPath));
+    try {
+      fs.ensureDirSync(podDstPath);
```

Comment:
I think it needs to have {recursive: true} if multiple parts of the path don't exist yet.

Reply:
I think it will work fine if we loose path.dirname from  fs.ensureDirSync(path.dirname(podDstPath)) and just keep
```js
fs.ensureDirSync(podDstPath);
```
since we need to create dir till podDstPath.  path.dirname will consider the last value in URI as file.

Reply:
your right. i forgot that graphviz pod creates a directory instead of a file. that's the proper fix

Reply:
So the reason it was set up in this try/catch fashion was due to the fact that `fs.ensureDirSync()` doesn't actually have an option to set `{ recursive: true }` - `path.dirname` was already removed from this current version of the pod, and it still breaks without it.

And as an additional note, the graphviz pod only creates a directory if it needs to. The actual output is a single file

Reply:
the current code still has dirname. see [here](https://github.com/dendronhq/dendron/blob/integ-publish/packages/pods-core/src/builtin/GraphvizPod.ts#L132:L133)

Reply:
Oh, yes - I was referring to this PR. The first thing I tried in order to fix this issue was removing `path.dirname` from `fs.ensureDirSync()`, but the method still failed when the target directory did not already exist